src/index.css

```diff
@@ -64,6 +64,7 @@ body,
 }
 
 body {
+	font-family: Frank Ruhl Libre;
```

Comment:
Looks like we still have some weird linting going on where this file is using spaces but this new line is using tabs. We can get it fixed later, just putting it out there